exporter/elasticsearchexporter/bulkindexer.go

```diff
@@ -463,18 +467,19 @@ func flushBulkIndexer(
 			))
 			tb.ElasticsearchDocsProcessed.Add(ctx, int64(itemsCount), attrSet)
 			tb.ElasticsearchBulkRequestsCount.Add(ctx, int64(1), attrSet)
+			tb.ElasticsearchBulkRequestsLatency.Record(ctx, latency, attrSet)
```

Comment:
Would be good to also set the `semconv.HTTPResponseStatusCode(http.StatusInternalServerError)` to align with always having the status code set.